control_protocol.md

```diff
@@ -330,17 +330,490 @@ A Component MUST also offer a list of all possibly callable methods in accordanc
+                },
+                "required": true
+            },
+            "description": "Compose a dictionary with the local directory."
+        },
+        {
+            "name": "get_component_names",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "clean_addresses",
+            "params": [
+                {
+                    "name": "expiration_time",
+                    "schema": {
+                        "type": "number"
+                    },
+                    "required": true
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            },
+            "description": "Clean all expired addresses from the directory.\n\n        :param float expiration_time: Expiration limit in s.\n        "
+        },
+    ],
+    "components": {
+        "schemas": {}
+    }
+}
+:::
+
+
+#### Actor
+
+An Actor is a {ref}`control_protocol.md#Component`.
+Additionally, it MUST offer the following methods.
+
+::: json
+{
+    "openrpc": "1.2.6",
+    "info": {
+        "title": "Actor",
+        "version": "0.1.0"
+    },
+    "methods": [
+        {
+            "name": "get_parameters",
+            "params": [
+                {
+                    "name": "parameters",
+                    "schema": {
+                        "anyOf": [
+                            {
+                                "type": "array",
+                                "items": {
+                                    "type": "string"
+                                }
+                            },
+                            {
+                                "type": "array",
+                                "items": {
+                                    "type": "string"
+                                }
+                            }
+                        ]
+                    },
+                    "required": true
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "object",
+                    "additionalProperties": true
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "set_parameters",
+            "params": [
+                {
+                    "name": "parameters",
+                    "schema": {
+                        "type": "object",
+                        "additionalProperties": true
+                    },
+                    "required": true
```

Comment:
In my understanding of #29, we would have `set_parameter`, that takes a parameter name and a value. This does not reflect that AFAICT.
How we deal with setting multiple at once I think we have not discussed yet -- maybe implement/schematize the single versions, first?

Reply:
That object should be a dictionary with key value pairs.